py/examples/picker_selection.py

```diff
@@ -14,14 +14,14 @@ async def serve(q: Q):
         ]
     else:
         q.page['example'] = ui.form_card(box='1 1 4 5', items=[
-            ui.picker(name='picker', label='Picker with initial values', choices=[
+            ui.picker(name='picker', label='Picker with initial values and max of 3 choices', choices=[
                 ui.choice(name='spam', label='Spam'),
                 ui.choice(name='eggs', label='Eggs'),
                 ui.choice(name='ham', label='Ham'),
                 ui.choice(name='cheese', label='Cheese'),
                 ui.choice(name='beans', label='Beans'),
                 ui.choice(name='toast', label='Toast'),
-            ], values=['spam', 'eggs']),
+            ], max_choices=3, values=['spam', 'eggs']),
```

Comment:
I am assuming this is a leftover from development. Please remove as these changes are not relevant to the feature this PR addresses.

Reply:
You are correct. I added `max_choices` to the example for the Cypress test I created to test the changes. Of course that was unnecessary, per your other observation! I've reverted all the changes to the example.